docs/releases/v1.1.0.rst

```diff
@@ -6,6 +6,11 @@
     -----
     .. current-class:: datatable.Frame
 
+    -[new] Class :class:`dt.FExpr` now has methods :meth:`.as_type() <dt.FExpr.as_type>`,
+      :meth:`.cov() <dt.FExpr.cov>`, and :meth:`.corr() <dt.FExpr.mean>`,
+      which behaves exactly as the equivalent base level functions :func:`dt.as_type()`,
+      :func:`dt.cov()`, and :func:`dt.corr()` respectively.
+
```

Comment:
This should probably be in the section "FExpr" below.

Also, please change `.. current-module:: datatable` into `.. current-class:: FExpr` there, then you won't have to write explicit address for each function. For example, ``:meth:`.corr()` `` should work just fine (and certainly better than having it redirected to the "mean()" method :)